src/System.CommandLine/ParseDiagramDirective.cs

```diff
@@ -6,25 +6,27 @@ namespace System.CommandLine
     /// Enables the use of the <c>[parse]</c> directive, which when specified on the command line will short 
     /// circuit normal command handling and display a diagram explaining the parse result for the command line input.
     /// </summary>
-    public sealed class ParseDirective : Directive
+    public sealed class ParseDiagramDirective : CliDirective
```

Comment:
The approved name is `DiagramDirective`. I am going to send a separate PR with a rename and changing `[parse]` to `[diagram]` just to start a discussion